relevanceai/operations_new/cluster/text/explainer/base.py

```diff
@@ -0,0 +1,268 @@
+"""Explainer
+"""
+import warnings
+import requests
+import numpy as np
+import itertools
+from typing import List, Callable
+from doc_utils import DocUtils
+
+
+class BaseExplainer(DocUtils):
+    """
+    Base class for vector
+    """
+
+    __name__ = ""
+
+    @classmethod
```

Comment:
why is this a `classmethod` ?

Reply:
In case this is needed outside of this class.